LayoutTest/TestCase/LYTLayoutTestCase.m

```diff
@@ -133,14 +133,22 @@ - (void)runBasicRecursiveViewTests:(UIView *)view {
     view = [LYTLayoutTestCase viewForSubviewTesting:view];
 
     [view lyt_recursivelyTraverseViewHierarchyWithStop:^(UIView *subview, BOOL *stopBranch) {
-        if ([LYTLayoutTestCase class:[subview class] includedInSet:self.viewClassesAllowingSubviewErrors] ||
-            subview.hidden) {
+         if (subview.hidden) {
+             // If the subview is hidden, let's not run any tests
             *stopBranch = YES;
```

Comment:
this won't stop the whole traversal, right? just won't test subviews of the current subview?